modules/gdnative/gdnative.h

```diff
@@ -43,7 +43,8 @@
 class GDNativeScriptData;
 class GDNativeLibrary;
 
-struct NativeLibrary {
+class NativeLibrary {
```

Comment:
Why did you make it a class?

It doesn't really change anything, but those types are just placeholders for data. Not everything needs to be OOP here 😄